lib/matplotlib/text.py

```diff
@@ -1165,7 +1166,10 @@ def set_x(self, x):
 
         ACCEPTS: float
         """
-        self._x = x
+        if x is None or np.isfinite(x):
```

Comment:
... do we accept None?  what would that even mean?  (in any case this is inconsistent with the ACCEPTS: float just above).